files/docker/node/addons/entrypoint.sh

```diff
@@ -1,37 +1,35 @@
 #!/bin/bash
 
-echo "NETWORK: $NETWORK";
-. ~/.bashrc
+trap 'killall -s SIGTERM cardano-node' SIGINT SIGTERM
+# "docker run --init" to enable the docker init proxy
+# To manually test: docker kill -s SIGTERM container
 
-export CNODE_HOME=/opt/cardano/cnode
-export CNODE_PORT=6000
+head -n 8 ~/.scripts/banner.txt
+
+. ~/.bashrc > /dev/null 2>&1
+
+echo "NETWORK: $NETWORK $POOL_NAME";
+
+[[ -z "${CNODE_HOME}" ]] && export CNODE_HOME=/opt/cardano/cnode 
+[[ -z "${CNODE_PORT}" ]] && export CNODE_PORT=6000
 
 echo "NODE: $HOSTNAME - Port:$CNODE_PORT - $POOL_NAME";
 cardano-node --version;
 
 sudo touch /etc/crontab /etc/cron.*/*
 sudo cron  > /dev/null 2>&1
-#sudo /etc/init.d/promtail start > /dev/null 2>&1
 
 dbsize=$(du -s ${CNODE_HOME}/db | awk '{print $1}')
-tnsizedb=$(du -s $CNODE_HOME/priv/testnet-db | awk '{print $1}')
-mnsizedb=$(du -s $CNODE_HOME/priv/mainnet-db | awk '{print $1}')
-
-if [[ $dbsize < $mnsizedb ]] && [[ $NETWORK == "mainnet" ]]; then
-cp -rf $CNODE_HOME/priv/mainnet-db ${CNODE_HOME}/db 2>/dev/null
-fi
+bksizedb=$(du -s $CNODE_HOME/priv/$NETWORK-db 2>/dev/null | awk '{print $1}')
 
-if [[ $dbsize < $tnsizedb ]] && [[ $NETWORK == "testnet" ]] ; then
-cp -rf $CNODE_HOME/priv/testnet-db ${CNODE_HOME}/db \ 2>/dev/null
-fi
-
-if [[ "${POOL_NAME}" ]] ; then 
-export POOL_DIR="$CNODE_HOME/priv/pool/$POOL_NAME"
-echo "POOL_DIR set to: $POOL_DIR" ;
+if [[ "$dbsize" -lt "$bksizedb" ]]; then
+cp -rf $CNODE_HOME/priv/$NETWORK-db/* ${CNODE_HOME}/db 2>/dev/null
```

Comment:
We found that data inside the `${CNODE_HOME}/db` can cause cardano-node to fail to start. @Scitz0 I believe relocated block db/logs to `${CNODE_HOME}/guild-db/blocklog`.

Reply:
I think I misunderstood this part initially. Looking it over it appears to be a snapshot deployment, not the blocklogs. So everything LGTM.